packages/arcgis-rest-request/src/request.ts

```diff
@@ -388,6 +388,7 @@ export function internalRequest(
         // encode the parameters into the query string
         const queryParams = encodeQueryString(params);
         // dont append a '?' unless parameters are actually present
+        /* istanbul ignore next */
         const urlWithQueryString =
           queryParams === "" ? url : url + "?" + encodeQueryString(params);
```

Comment:
I think you just want to ignore the true case then, like this?

```suggestion
        const urlWithQueryString =
queryParams === "" ? /* istanbul ignore next */ url : url + "?" + encodeQueryString(params);
```